unreal_pak/src/buf_ext.rs

```diff
@@ -1,11 +1,11 @@
 use std::io::{Read, Write};
 
-use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
+use byteorder::{ReadBytesExt, WriteBytesExt, LE};
 
 use crate::error::PakError;
 
 pub trait BufReaderExt {
-    fn read_fstring(&mut self) -> Result<Option<String>, PakError>;
+    fn read_fstring(&mut self) -> Result<String, PakError>;
```

Comment:
maybe unify this function between uasset/unrealpak?

Reply:
going to do that after the PR is merged